diff --git a/cmake/KleidiAI.cmake b/cmake/KleidiAI.cmake
index 52ce998..7d7af00 100644
--- a/cmake/KleidiAI.cmake
+++ b/cmake/KleidiAI.cmake
@@ -30,7 +30,7 @@ function (download_kleidiai_and_collect_sources)
     set(KLEIDIAI_BUILD_TESTS OFF)
 
     set(KLEIDIAI_COMMIT_SHA "1.16.0")
-    set(KLEIDIAI_DOWNLOAD_URL "https://github.com/ARM-software/kleidiai/archive/refs/tags/v${KLEIDIAI_COMMIT_SHA}.tar.gz")
+    set(KLEIDIAI_DOWNLOAD_URL "https://ghfast.top/https://github.com/ARM-software/kleidiai/archive/refs/tags/v${KLEIDIAI_COMMIT_SHA}.tar.gz")
     set(KLEIDIAI_ARCHIVE_MD5 "0a9e9008adb6031f9e8cf70dff4a3321")
 
     set(_kleidiai_src_dir "")
